pkg/gds/admin.go

```diff
@@ -136,17 +136,37 @@ func (s *Admin) setupRoutes() (err error) {
 
 	// Add the v2 API routes
 	v2 := s.router.Group("/v2")
+	v2.POST("/authenticate", s.Authenticate)
+	v2.POST("/reauthenticate", admin.DoubleCookie(), s.Reauthenticate)
 	v2.GET("/status", s.Status)
 	v2.GET("/summary", s.Summary)
 	v2.GET("/autocomplete", s.Autocomplete)
 	v2.GET("/vasps", s.ListVASPs)
 	v2.GET("/vasps/:vaspID", s.RetrieveVASP)
-	v2.POST("/vasps/:vaspID/review", s.Review)
-	v2.POST("/vasps/:vaspID/resend", s.Resend)
+	v2.POST("/vasps/:vaspID/review", admin.DoubleCookie(), s.Review)
+	v2.POST("/vasps/:vaspID/resend", admin.DoubleCookie(), s.Resend)
 
 	return nil
 }
 
+func (s *Admin) Authenticate(c *gin.Context) {
+	if err := admin.SetDoubleCookieTokens(c, time.Now().Add(time.Minute*10).Unix()); err != nil {
```

Comment:
Excellent question - I think the cookies should have the same expiration as the refresh token since the Authenticate and Reauthenticate methods are the things that set the cookies.  When these methods create the tokens they will use the `NotBefore` property and pass that to the `SetDoubleCookieTokens` function. The configuration for the duration of the tokens is currently a global variable set here: https://github.com/trisacrypto/directory/blob/main/pkg/gds/tokens/tokens.go#L17-L22